BlueShift-iOS-SDK/BlueshiftEventAnalyticsHelper.m

```diff
@@ -114,17 +114,12 @@ +(BOOL)isMarkInAppAsOpen:(NSDictionary*)userInfo {
 
 + (BOOL) isInAppMessagePayload: (NSDictionary*)userInfo {
```

Comment:
as we are checking if this is a silent push message, please suggest a better name to avoid future confusion.

Reply:
please do the same for `createInAppNotification` method inside `BlueShift.m`